lib/TPP/ConvertCheckToFunc.cpp

```diff
@@ -0,0 +1,172 @@
+//===- ConvertCheckToFunc.cpp -------------------------------------*- C++-*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "TPP/Dialect/Check/CheckOps.h"
+#include "TPP/Passes.h"
+#include "TPP/Transforms.h"
```

Comment:
not needed.

Reply:
Method populateCheckToFuncPatterns is defined in TPP/Transforms.h. Do you recommend moving it elsewhere?

Reply:
Ah, then it is fine